### DISCUSSION TITLE
how do i disable videos playing from the homepage?

### DISCUSSION THREAD

**Discussion by TheCheddarCheese** [PoomSmart/YTABConfig#89] [Q&A]
![image](https://github.com/PoomSmart/YTABConfig/assets/124603704/32cb2a10-81a7-4772-b310-f9f3e3da4c41)

**Comment by VoidAny** [comment]
Go to regular YouTube settings>General>Playback in feeds and turn it off

**Comment by Aholicknight** [comment]
> Go to regular YouTube settings>General>Playback in feeds and turn it off

This was removed. Is there a flag for this still?

**Reply by Harishwarrior** [reply]
![image](https://github.com/user-attachments/assets/f884af96-7b88-48ed-8877-840b8e525cbd)

Try this

**Reply by Aholicknight** [reply]
This works!
